wdtk-distribution/pom.xml

```diff
@@ -34,14 +41,25 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-assembly-plugin</artifactId>
 				<version>2.4</version>
+				<configuration>
+					<descriptorRefs>
+						<descriptorRef>jar-with-dependencies</descriptorRef>
+					</descriptorRefs>
+					<appendAssemblyId>false</appendAssemblyId>
+				</configuration>
 				<executions>
 					<execution>
-						<id>create-binary-jar</id>
+						<id>make-assembly</id>
```

Comment:
What's the reason for this change? Could this break anything that worked before?